c_src/h3.c

```diff
@@ -633,11 +633,16 @@ erl_grid_distance(ErlNifEnv * env, int argc, const ERL_NIF_TERM argv[])
     }
 
     int64_t result = h3Distance(h3idx_src, h3idx_dest);
+    if (result < 0)
```

Comment:
Nice catch! If I get around to the next batch of edge functions, I think it could be useful to have an h3 _edge_ helper function.

Reply:
Yeah, when we get to it, helpers will keep the code DRY